source/what-we-do/index.html.md.erb

```diff
@@ -3,4 +3,44 @@ title: What we do
 weight: 3
 ---
 
-# What we do
\ No newline at end of file
+# What we do
+
+We maintain the GOV.UK Design System and related tools.
+
+## Our products
+
+### Actively maintained
+
+- [GOV.UK Design System Website] - guidance for making government services consistent with GOV.UK. The code is maintained in the [govuk-design-system GitHub repository].
+- [GOV.UK Frontend] - a code library of GOV.UK Design System components.
+- [GOV.UK Frontend Docs] - Technical documentation for installing and using GOV.UK Frontend. The code is maintained in the [govuk-frontend-docs GitHub repository].
+- [Community backlog] - a backlog of user-submitted component and pattern suggestions for inclusion into the Design System.
+- [stylelint-config-gds] - a stylelint configuration for SCSS and CSS files across GDS.
+
+### Security patches only
+
+- [Accessible Autocomplete] - a component that helps users choose answers from a provided list
+
+### Legacy products
+
+- [GOV.UK Elements]
+- [GOV.UK Frontend Toolkit]
+- [GOV.UK Template]
+
+### No longer supported
+
+- [GOV.UK Prototype Kit] - The GOV.UK Prototype Kit team is responsible for maintenance and support.
```

Comment:
This now needs to be updated, is there an official line that can be inserted? Where does this fit in "actively maintained", "security patches only", etc?

Reply:
Prototype Kit is in "maintenance mode" — it gets bugfixes, security patches, etc. but won't be getting new features.

Maybe this points to needing clearer headings. "Actively developed" instead of "Actively maintained"?

Reply:
Kelly outlined the maintenance and support we can provide here: https://github.com/alphagov/govuk-prototype-kit/issues/2389

In case that's helpful for amending the content.

Reply:
Had a go at this - switched out "Actively developed" for "actively maintained" and then some clarifications for Accessible Autocomplete and Prototype Kit.

Reply:
Looks good!